tests/xharness/Execution/MLaunchArguments.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Xharness.Execution {
+	// mlaunch is really important and has a lot of arguments that are known but
+	// used to be passed as strings. This class allows to add arguments without
+	// knowing the exact string and will also validate that an argument that
+	// needs a value does contain the valuw
+	public enum MlaunchArgumentType {
```

Comment:
>MlaunchArgumentType [](start = 13, length = 19)

I checked all the usages of Harness.MlaunchPath and it seems there are way more many usages and more arguments.

Is the plan to migrate those piece by piece later or they are left out unintentionally?

Reply:
Correct, jus moving the ones I'm touching atm.